notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<p>For better readability of the code, especially when using different third party libraries, I would suggest to use </p><p> </p><p><code>import dynamics</code> </p><p> </p><p>and then later use the <code>dynamics</code> functions and classes like</p><p><code>dynamics.xyz</code></p><p> </p><p>Same for all other import statements that end in wildcard <code>*</code>  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thank you! Import statements are correctly declared through the notebook.</p>

Reply:
<p>closing conversation, resolved</p>